R/render.R

```diff
@@ -210,14 +210,16 @@ tsukuyomi = function(...) moon_reader(...)
 #' @param moon The input Rmd file path (if missing and in RStudio, the current
 #'   active document is used).
 #' @param cast_from The root directory of the server.
+#' @param params List of named parameters that override custom params
+#'   specified within the YAML front-matter.
```

Comment:
I usually document such params as "Passed to [rmarkdown::render()]" rather than using a shortened version of the original param description.